### DISCUSSION TITLE
Welcome to simple_sheet_music Discussions!

### DISCUSSION THREAD

**Discussion by tomoyu719** [tomoyu719/simple_sheet_music#23] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by beattizzle** [comment]
Good morning,

My name is Robert and I am working on a music theory app geared towards guitar players.  I'm currently using this library to generate flashcards that help with memorization of key signatures, but plan to use it for other things in the future.

I am super grateful for the people who work on this project!  I've only submitted a couple of minor bug fixes thus far, but am willing to help out more as I get more familiar with the code.

Cheers.

**Reply by tomoyu719** [reply]
Hi, Robert,

Good morning! Pleased to meet you. This library was created for people like you. It is still buggy and has limited functionality, but we plan to improve it.

Your contribution is sincerely appreciated. If you have any questions, don't hesitate to let us know.

We look forward to seeing your app develop!

Cheers.
Tomoyuki Kitano
